WorkflowUI/Tests/ViewControllerDescriptionTests.swift

```diff
@@ -54,9 +54,9 @@
 
             final class SubclassViewController: BlankViewController {}
 
-            // We only update exact type matches, not subclasses
+            // We can update subclasses too, as long as they pass an "is/as?" check.
             let subclassViewController = SubclassViewController()
-            XCTAssertFalse(description.canUpdate(viewController: subclassViewController))
```

Comment:
Can we make sure there's also a test that has a `XCTAssertFalse` for this? As well, want to add a test case for your actual use case (eg, a `UIViewController & FooThing` protocol)?

Reply:
Ah cool, sorry, missed that one in my peruse of the file!